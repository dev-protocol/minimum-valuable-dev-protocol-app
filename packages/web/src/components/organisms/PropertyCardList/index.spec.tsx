import React from 'react'
import { render } from '@testing-library/react'
import { PropertyCardList } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/utility/gql-hooks-wrapper')
jest.mock('@dev/graphql')

describe(`${PropertyCardList.name}`, () => {
  test('Snapshot', () => {
    const component = render(<PropertyCardList />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
