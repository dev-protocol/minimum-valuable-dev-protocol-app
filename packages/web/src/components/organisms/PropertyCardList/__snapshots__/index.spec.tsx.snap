// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyCardList Snapshot 1`] = `
<body>
  <div>
    <div
      style="flex-grow: 1; max-width: 100vw;"
    >
      <div
        class="sc-190q6or-1 eQYYSh"
      >
        <h2
          class="sc-190q6or-0 kPXJaM"
        >
          Asset Pools
        </h2>
        <div
          class="sc-13ny1rq-0 jXyMXa"
        >
          <div
            class="sc-11akcse-0 krFORN"
          >
            <div
              class="sc-11akcse-1 kSoTqI"
            >
              <span
                aria-label="search"
                class="anticon anticon-search"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="search"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                  />
                </svg>
              </span>
            </div>
            <input
              class="sc-11akcse-3 aNMHg"
              data-cy="SearchbarInput"
              placeholder="Search for a project"
              value=""
            />
          </div>
        </div>
        <div
          class="sc-190q6or-4 fHetqn"
        >
          <div
            style="flex-grow: 1;"
          >
            <div
              class=" css-2b097c-container"
            >
              <span
                aria-atomic="false"
                aria-live="polite"
                aria-relevant="additions text"
                class="css-1f43avz-a11yText-A11yText"
              />
              <div
                class=" css-yk16xz-control"
              >
                <div
                  class=" css-g1d714-ValueContainer"
                >
                  <div
                    class=" css-1uccc91-singleValue"
                  >
                    Most Recent
                  </div>
                  <div
                    class="css-b8ldur-Input"
                  >
                    <div
                      class=""
                      style="display: inline-block;"
                    >
                      <input
                        aria-autocomplete="list"
                        autocapitalize="none"
                        autocomplete="off"
                        autocorrect="off"
                        id="react-select-2-input"
                        spellcheck="false"
                        style="box-sizing: content-box; width: 2px; border: 0px; opacity: 1; outline: 0; padding: 0px;"
                        tabindex="0"
                        type="text"
                        value=""
                      />
                      <div
                        style="position: absolute; top: 0px; left: 0px; visibility: hidden; height: 0px; overflow: scroll; white-space: pre; font-family: -webkit-small-control; letter-spacing: normal; text-transform: none;"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class=" css-1hb7zxy-IndicatorsContainer"
                >
                  <div
                    aria-hidden="true"
                    class=" css-tlfecz-indicatorContainer"
                  >
                    <svg
                      aria-hidden="true"
                      class="css-tj5bde-Svg"
                      focusable="false"
                      height="20"
                      viewBox="0 0 20 20"
                      width="20"
                    >
                      <path
                        d="M14.348 14.849c-0.469 0.469-1.229 0.469-1.697 0l-2.651-3.030-2.651 3.029c-0.469 0.469-1.229 0.469-1.697 0-0.469-0.469-0.469-1.229 0-1.697l2.758-3.15-2.759-3.152c-0.469-0.469-0.469-1.228 0-1.697s1.228-0.469 1.697 0l2.652 3.031 2.651-3.031c0.469-0.469 1.228-0.469 1.697 0s0.469 1.229 0 1.697l-2.758 3.152 2.758 3.15c0.469 0.469 0.469 1.229 0 1.698z"
                      />
                    </svg>
                  </div>
                  <span
                    class=" css-1okebmr-indicatorSeparator"
                  />
                  <div
                    aria-hidden="true"
                    class=" css-tlfecz-indicatorContainer"
                  >
                    <svg
                      aria-hidden="true"
                      class="css-tj5bde-Svg"
                      focusable="false"
                      height="20"
                      viewBox="0 0 20 20"
                      width="20"
                    >
                      <path
                        d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="dkx7pb-2 kpSDwP"
          >
            <button
              class="dkx7pb-0 bYegHx"
            >
              <div
                class="dkx7pb-1 fqMBMb"
              />
              <span>
                DEV
              </span>
            </button>
            <button
              class="dkx7pb-0 dYFXL"
            >
              <div
                class="dkx7pb-1 fKortD"
              />
              <span>
                USD
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="sc-190q6or-3 hSHfbB"
      >
        <a
          href="/"
        >
          All Pools
        </a>
        <a
          href="/?tag=GitHub"
          style="color: rgb(201, 201, 201);"
        >
          GitHub
        </a>
        <a
          href="/?tag=npm"
          style="color: rgb(201, 201, 201);"
        >
          npm
        </a>
        <a
          href="/?tag=Creators"
          style="color: rgb(201, 201, 201);"
        >
          Creators
        </a>
      </div>
      <div
        class="oxa8xu-0 sc-190q6or-2 fyzTpI bcjFbf"
      >
        <div
          class="oxa8xu-0 k29mxm-0 fyzTpI kZabSJ"
        >
          <div
            class="k29mxm-15 dyNzQ"
          />
          <div
            class="oxa8xu-0 k29mxm-8 fyzTpI eitDqS"
          >
            <span
              class="k29mxm-2 iMzPpN"
            >
              authentication-id
            </span>
            <span
              class="k29mxm-13 cJJZoJ"
            >
              Stake DEV tokens to provide funding for OSS projects so that they can maintain development.
            </span>
            <div
              class="k29mxm-12 liExmL"
            >
              <div
                class="sc-17om263-0 hTjxnI"
              >
                <div
                  class="sc-17om263-1 cwMCjn"
                  size="60"
                />
              </div>
              <div
                class="k29mxm-14 gfMhAn"
              >
                <span
                  style="font-weight: lighter;"
                >
                  Creator
                </span>
                <span
                  style="color: rgb(26, 201, 252);"
                />
                <span>
                  0
                   Karma
                </span>
              </div>
            </div>
            <div
              class="k29mxm-1 eDHAFi"
            >
              <div
                class="k29mxm-3 gzeqFX"
              >
                <span>
                  5,000
                   
                </span>
                <span
                  class="k29mxm-11 tCjlQ"
                >
                  Your stake
                </span>
              </div>
              <div
                class="k29mxm-5 jxFlpl"
              >
                <span>
                  5,000
                   
                </span>
                <span
                  class="k29mxm-11 tCjlQ"
                >
                  Your reward
                </span>
              </div>
              <div
                class="k29mxm-4 lPEPx"
              >
                <span>
                  10,000
                   
                </span>
                <span
                  class="k29mxm-11 tCjlQ"
                >
                  Total staked
                </span>
              </div>
              <div
                class="k29mxm-6 jTXFEC"
              >
                <span>
                  10,000
                   
                </span>
                <span
                  class="k29mxm-11 tCjlQ"
                >
                  Creator reward
                </span>
              </div>
            </div>
          </div>
          <div
            class="k29mxm-7 bqLKkp"
          >
            <button
              class="k29mxm-9 lkZTRi"
            >
              Stake
            </button>
            <button
              class="k29mxm-10 dBfhGy"
            >
              Withdraw
            </button>
          </div>
        </div>
      </div>
      <div
        class="sc-190q6or-5 NhEcr"
      >
        <ul
          class="ant-pagination"
          unselectable="unselectable"
        >
          <li
            aria-disabled="true"
            class="ant-pagination-prev ant-pagination-disabled"
            title="Previous Page"
          >
            <button
              class="ant-pagination-item-link"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <span
                aria-label="left"
                class="anticon anticon-left"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="left"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                  />
                </svg>
              </span>
            </button>
          </li>
          <li
            class="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
            tabindex="0"
            title="1"
          >
            <a
              rel="nofollow"
            >
              1
            </a>
          </li>
          <li
            class="ant-pagination-item ant-pagination-item-2"
            tabindex="0"
            title="2"
          >
            <a
              rel="nofollow"
            >
              2
            </a>
          </li>
          <li
            class="ant-pagination-item ant-pagination-item-3"
            tabindex="0"
            title="3"
          >
            <a
              rel="nofollow"
            >
              3
            </a>
          </li>
          <li
            class="ant-pagination-item ant-pagination-item-4"
            tabindex="0"
            title="4"
          >
            <a
              rel="nofollow"
            >
              4
            </a>
          </li>
          <li
            class="ant-pagination-item ant-pagination-item-5"
            tabindex="0"
            title="5"
          >
            <a
              rel="nofollow"
            >
              5
            </a>
          </li>
          <li
            class="ant-pagination-item ant-pagination-item-6"
            tabindex="0"
            title="6"
          >
            <a
              rel="nofollow"
            >
              6
            </a>
          </li>
          <li
            aria-disabled="false"
            class="ant-pagination-next"
            tabindex="0"
            title="Next Page"
          >
            <button
              class="ant-pagination-item-link"
              tabindex="-1"
              type="button"
            >
              <span
                aria-label="right"
                class="anticon anticon-right"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="right"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                  />
                </svg>
              </span>
            </button>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div>
    <div
      class="ant-message"
    >
      <div>
        <div
          class="ant-message-notice"
        >
          <div
            class="ant-message-notice-content"
          >
            <div
              class="ant-message-custom-content ant-message-error"
            >
              <span
                aria-label="close-circle"
                class="anticon anticon-close-circle"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="close-circle"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
                  />
                </svg>
              </span>
              <span>
                fetch is not defined
              </span>
            </div>
          </div>
        </div>
        <div
          class="ant-message-notice"
        >
          <div
            class="ant-message-notice-content"
          >
            <div
              class="ant-message-custom-content ant-message-error"
            >
              <span
                aria-label="close-circle"
                class="anticon anticon-close-circle"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="close-circle"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
                  />
                </svg>
              </span>
              <span>
                fetch is not defined
              </span>
            </div>
          </div>
        </div>
        <div
          class="ant-message-notice"
        >
          <div
            class="ant-message-notice-content"
          >
            <div
              class="ant-message-custom-content ant-message-error"
            >
              <span
                aria-label="close-circle"
                class="anticon anticon-close-circle"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="close-circle"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
                  />
                </svg>
              </span>
              <span>
                fetch is not defined
              </span>
            </div>
          </div>
        </div>
        <div
          class="ant-message-notice"
        >
          <div
            class="ant-message-notice-content"
          >
            <div
              class="ant-message-custom-content ant-message-error"
            >
              <span
                aria-label="close-circle"
                class="anticon anticon-close-circle"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="close-circle"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
                  />
                </svg>
              </span>
              <span>
                fetch is not defined
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      class="ant-message"
    >
      <div />
    </div>
  </div>
  <div>
    <div
      class="ant-message"
    >
      <div />
    </div>
  </div>
  <div>
    <div
      class="ant-message"
    >
      <div />
    </div>
  </div>
</body>
`;
